core-bundle/src/Resources/contao/modules/ModuleQuicklink.php

```diff
@@ -91,13 +91,6 @@ protected function compile()
 		{
 			$objSubpage->loadDetails();
 
-			// Hide the page if it is not protected and only visible to guests (backwards compatibility)
-			if ($objSubpage->guests && !$objSubpage->protected && $isMember)
```

Comment:
this should be a separate PR (including `tl_page.guests` and migration)

Reply:
Reverted in 965ab6aa605345f3e860857032cebdeb8a3bdd8f.